puppet/zulip/manifests/postgres_common.pp

```diff
@@ -7,8 +7,6 @@
                         "ptop",
                         # Python modules used in our monitoring/worker threads
                         "python-gevent",
-                        "python-tz",
-                        "python-dateutil",
```

Comment:
same comment as above, here.  These two are used by:

puppet/zulip/files/nagios_plugins/zulip_postgres_common/check_postgres_backup

which runs on the database server.